polyply/src/random_walk.py

```diff
@@ -65,54 +66,22 @@ def _is_overlap(meta_molecule, point, tol, fudge=1):
     -------
     bool
     """
-    for node in meta_molecule:
-        try:
-            coord = meta_molecule.nodes[node]["position"]
-        except KeyError:
-            continue
-
-        if np.linalg.norm(coord - point) < tol * fudge:
-            return True
-
+    red_pos = positions[positions[:, 0] != np.inf]
```

Comment:
red?
And I would use np.isfinite. That also catches -np.inf, and nan

Reply:
This is outdated but will come back elsewhere np.isfinite is too slow it does too many checks and the values cannot be non or -np.inf by definition within the program. 

red as in reduced positions

Reply:
Needs a comment then